platformio.ini

```diff
@@ -184,3 +186,78 @@ build_flags =
 
   ; Reduce loop task stack size (only works on newer IDF Arduino core)
   ; -DARDUINO_LOOP_STACK_SIZE=2048
+
+  [env:view_s3_stspin]
```

Comment:
Can you remove the indent here? 
The CIO check should pass after you fix this.

Reply:
Sure, corrected.